partitioned-table.md

```diff
@@ -579,7 +579,19 @@ You can see that the inserted record `(NULL, 'mothra')` falls into the same part
 
 ## Partition management
 
-You can add, drop, merge, split, redefine partitions by using `ALTER TABLE` statements.
+For `LIST` and `RANGE` partitioned tables, you can add and drop partitions using the `ALTER TABLE <table name> ADD PARTITION (<partition specification>)` or `ALTER TABLE <table name> DROP PARTITION <list of partitions>` statement.
+
+For `HASH` partitioned tables, `COALESCE PARTITION` and `ADD PARTITION` is not yet supported.
```

Comment:
```suggestion
For `HASH` partitioned tables, `COALESCE PARTITION` and `ADD PARTITION` are not yet supported.
```

Reply:
Updated, thanks for spotting the bad grammar.